config/routes.rb

```diff
@@ -6,14 +6,30 @@
   # resources :task
   root 'task#index'
 
```

Comment:
A lot of these routes are not set up as RESTful. Be cautious when using routes that are not RESTful because this could potentially cause issues later - especially if you want to use the default `resources`